### DISCUSSION TITLE
Possible tutorial

### DISCUSSION THREAD

**Discussion by Shrewtl** [mmdanggg2/D3D9DrvRTX#10] [General]
Hi there,

I saw an article promoting what you’ve managed to do and it looks amazing! I’m not much of a computer whizz and was wondering if it’s possible for you or someone to do a video showing how to install the files and make the game look fresh and crisp? 

I’m a massive Harry Potter fan and I’d love to promote it on my stream for other people 😊

Thank you so much in advance!

**Comment by mmdanggg2** [comment]
Just posted a video tutorial to help get people started here: https://youtu.be/XEe-pyZ3J9g
